dist/stylesheets/style.css

```diff
@@ -1,3 +1,22 @@
 body {
   background-color: magenta;
 }
+
+.divGreen {
```

Comment:
This is great. Are you planning on transferring over the names of the colors to the feature when we are developing?